### PR TITLE
Order css result

### PR DESCRIPTION
**Pull request by Johann-S**

### CODE REVIEW

**Comment by XhmikosR** [comment]
Reboot isn't included with every CSS component here, right?

**Comment by Johann-S** [comment]
That's not possible I included it everywhere

File: src/util.js [hunk 1]
```diff
@@ -21,9 +21,107 @@ const formatList = (list) => {
...
+    'code',
+    'gird',
+    'tables',
+    'forms',
+    'buttons',
+    'transitions',
+    'dropdown',
+    'button-group',
+    'input-group',
+    'custom-form',
```
Comment (MartijnCuppens) [inline src/util.js]: Shouldn't this be _custom-form**s**_?

Comment (Johann-S) [reply src/util.js]: Yep you're right but it works like that too, I'll change that 👍

**Comment by XhmikosR** [comment]
Index of :p

**Comment by Johann-S** [comment]
Maybe one day to please you @XhmikosR I'll do that : 
```js
if (!Array.prototype.include) {
  Array.prototype.include = element => {
    return this.indexOf(element) !== -1
  }
}
```
🤣

File: src/util.js (hunk 1 above)
Comment (MartijnCuppens) [reply src/util.js]: I haven't tested this, I was just comparing the list order with the Bootstraps order and noticed this 🙂

**Comment by XhmikosR** [comment]
Looks good from a quick testing. Maybe wait for another approval just to be safe.

**Comment by Johann-S** [comment]
Ok so we'll wait for the approval of @MartijnCuppens

**Pull request merged**